torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -818,6 +818,17 @@ RegisterOperators reg(
              "aten::__contains__.str_list(str[] l, str item) -> bool"),
          listContains<std::string>,
          aliasAnalysisFromSchema()),
+     OperatorGenerator(
```

Comment:
Another way to support `__contains__` would be to use our mechanism to register operators that desguar to other operators in `builtin_functions.cpp`.  I think you would just add to the ops section here https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/frontend/builtin_functions.cpp#L65

We haven't been as judicious as we could have been in composing builtins in terms of other builtins, but I think it's generally a good idea to have less operators because they contribute to mobile size, and having fewer operators would make it easier to do things like do codegen for builtin python types down the road.